src/core/qgis.cpp

```diff
@@ -33,6 +33,9 @@
 #include <geos_c.h>
 #include <ogr_api.h>
 
+#define xstr(x) str(x)
```

Comment:
This would have been THE opportunity to call this `strk`

Reply:
xstr is a common name for such macro, if you search around :)
Do I need approval on the PR to merge ? Lost track of policy changes, if I do, could you please take the opportunity to approve it ? :)

Reply:
Yep, you need, will do as soon as I get a reply or patch for the other comment ;-)